xml/System.Reflection/MemberInfo.xml

```diff
@@ -591,7 +591,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Is MemberInfo part of an Assembly held in a collectible AssemblyLoadContext?</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value><see langword="true" /> if the <see cref="T:System.Reflection.MemberInfo" /> is part of an assembly held in a collectible assembly load context; otherwise, <see langword="false" />.</value>
```